### DISCUSSION TITLE
Issues tab is missing, so here is an issue we've found:

### DISCUSSION THREAD

**Discussion by ofluffydev** [rhinstaller/anaconda#6071] [General]
When trying to use Rufus:

The installer cannot continue due to a critical error: reading information about the computer failed.
org.fedoraproject.anaconda.error: one of the hex, bytes or int arguments must be given.

**Comment by jkonecny12** [comment]
Please use a [bugzilla](https://bugzilla.redhat.com/) to file bugs. Also please attach logs from the /tmp/*log in the installation environment.
